awx/ui_next/src/screens/Template/TemplateList/TemplateList.jsx

```diff
@@ -113,11 +117,11 @@ class TemplatesList extends Component {
     if (cachedActions) {
       optionsPromise = Promise.resolve({ data: { actions: cachedActions } });
     } else {
-      optionsPromise = JobTemplatesAPI.readOptions();
+      optionsPromise = UnifiedJobTemplatesAPI.readOptions();
     }
 
     const promises = Promise.all([
-      JobTemplatesAPI.read(params),
+      UnifiedJobTemplatesAPI.read(params),
```

Comment:
The template list add button is hidden. As a superuser, I'm only getting the action option to GET, there is no POST. I wonder if this is related to rebuilding with the latest downstream fixes this morning.

Reply:
See https://github.com/ansible/awx/pull/5379/commits/f439f15eb057d0594b62aca9afc3bd9ee954a4e5

<img width="1666" alt="Screen Shot 2019-12-02 at 2 52 37 PM" src="https://user-images.githubusercontent.com/9889020/69990311-687ff680-1513-11ea-8617-fa503fadeae0.png">